src/components/OSCALLoader.js

```diff
@@ -192,12 +224,31 @@ export default function OSCALLoader(props) {
   if (!isLoaded) {
     result = <CircularProgress />;
   } else if (oscalUrl) {
-    result = props.renderer(
-      isRestMode,
-      oscalData,
-      oscalUrl,
-      onResolutionComplete,
-      handleRestPatch
+    result = isRestMode ? (
```

Comment:
Per the acceptance criteria in EasyDynamics/easygrc#23, we need to be able to toggle source editing on and off.  The AC currently says source should be off by default but I think I'm going to change that.

Perhaps we could use the `Code` or `Source` material icon as the toggle.